abc.templates/README.md

```diff
@@ -0,0 +1,18 @@
+# GitHub Token Minter
+
+## Installation with abc CLI
+
+1. Render infrastructure template and actuate terraform.
+```shell
+abc templates render \
+--dest infra/github-token-minter \
+--input=automation_service_account_member=<SERVICE_ACCOUNT> \
+--input=project_id=<PROJECT_ID> \
+--input=domain=<DOMAIN> \
+--input=terraform_state_bucket=<TERRAFORM_STATE_BUCKET> \
+--input=terraform_state_prefix=<TERRAFORM_STATE_PREFIX> \
+--input=github_owner_id=<GITHUB_ORG_ID> \
```

Comment:
I believe per Seths previous comment these input flags should be single dash (-) not double (--)

Reply:
got it, https://github.com/abcxyz/abc#flags will have to be updated as well